bitnami/dataplatform-bp1/values.yaml

```diff
@@ -1002,61 +1106,62 @@ wavefront:
       enableRuntimeConfigs: true
       ## @param wavefront.collector.discovery.config [array] Configuration for rules based auto-discovery
       ##
-      config:
+      ## Example:
+      ## config:
         ## auto-discover kafka-exporter
         ##
```

Comment:
I would remove this because you are breaking the example. The `name` is self explanatory. And the same with the sections below, so the example can be just copy pasted